recipe/meta.yaml

```diff
@@ -12,22 +12,21 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
 
 requirements:
   host:
     - python >=3.6
```

Comment:
do not use >=3.6 in host for python.  Instead add a skip block for it ('skip: True. # [py<36]')

Reply:
Got it! Went ahead and made this amendment. Thanks!